### PR TITLE
Add an original poem about GitHub by Coach Dave

### PR DESCRIPTION
**Pull request by davidbtucker**
Here's my first poem!

### CODE REVIEW

File: coach-dave-roses.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Roses are red
+Violets are blue
+I like using GitHub
```
Comment (jkschulz) [inline coach-dave-roses.md]: Do you only "like" it?

**Review by jkschulz — CHANGES_REQUESTED** [review]
Can you please add punctuation?

**Comment by davidbtucker** [comment]
> Can you please add punctuation?

Nope, I like it the way it is.

File: coach-dave-roses.md (hunk 1 above)
Comment (davidbtucker) [reply coach-dave-roses.md]: Changed "like" to "love".

**Review by jkschulz — APPROVED** [review]

**Pull request merged**